src/CommonITILObject.php

```diff
@@ -6929,7 +6929,7 @@ public function getTimelineItemtypes(): array
                 if (is_callable($hook_itemtypes)) {
                     $hook_itemtypes = $hook_itemtypes(['item' => $this]);
```

Comment:
`$hook_itemtypes` is used initially to store the callable, then is replaced by callable result, maybe it should be renamed on loop declaration.
```diff
- foreach ($PLUGIN_HOOKS[Hooks::TIMELINE_ANSWER_ACTIONS] as $plugin => $hook_itemtypes) {
+ foreach ($PLUGIN_HOOKS[Hooks::TIMELINE_ANSWER_ACTIONS] as $plugin => $hook_callable) {
```